lib/highlight.js

```diff
@@ -47,6 +47,9 @@ module.exports = (stream) => {
     if (stream.getColorDepth() >= 8) level = 2;
     if (stream.getColorDepth() >= 24) level = 3;
   }
-  const colorSheet = sheet(new chalk.Instance({ level }));
-  return (s) => emphasize.highlight('js', s, colorSheet).value;
+  const chalkInstance = new chalk.Instance({ level });
+  const colorSheet = sheet(chalkInstance);
+  const highlight = (s) => emphasize.highlight('js', s, colorSheet).value;
+  highlight.colorizeMatchingBracket = (s) => chalkInstance.bgBlue(s);
```

Comment:
This is a bit more visible than what the legacy shell does, this adds a background instead of a color on the bracket. After playing around with both a bit, I like this more because it’s more obvious and not particularly intrusive, but feel free to disagree and suggest something else (or just the legacy shell’s behavior) :)